Jenkinsfile

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Contiki Applications
+ *
+ */
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '5')),
+    parameters([
+        stringParam(defaultValue: '26', description: 'Set 6LoWPAN Channel', name: 'CHANNEL'),
+        stringParam(defaultValue: '0xabcd', description: 'Set 6LoWPAN PAN ID', name: 'PAN_ID'),
+        stringParam(defaultValue: '1', description: 'Set Serial port to use for console', name: 'PADS'),
+    ])
+])
+
+node('docker && imgtec') {
+    def docker_image
+    docker_image = docker.image "imgtec/creator-builder:latest" // TODO for now using the creator one
+    docker_image.inside {
+        stage('Prepare') {
+            checkout([$class: 'GitSCM',
+                userRemoteConfigs: scm.userRemoteConfigs,
+                branches: scm.branches,
+                doGenerateSubmoduleConfigurations: scm.doGenerateSubmoduleConfigurations,
+                submoduleCfg: scm.submoduleCfg,
+                browser: scm.browser,
+                gitTool: scm.gitTool,
+                extensions: scm.extensions + [
+                    [$class: 'CleanCheckout'],
+                    [$class: 'PruneStaleBranch'],
+                ],
+            ])
+        }
+        stage('Build') {
+            sh 'git submodule init "dev/ca8210" && git submodule update'
+            sh "cd examples/hello-world && make TARGET=mikro-e USE_CA8210=1 CHANNEL=${params.CHANNEL} PAN_ID=${params.PAN_ID} USE_SERIAL_PADS=${params.PADS}"
+            sh 'cd examples/hello-world && xc32-bin2hex hello-world.mikro-e && mv hello-world.hex ${WORKSPACE}/'
+            sh 'cd examples/ && git clone https://github.com/mtusnio/LetMeCreateIoT.git'
```

Comment:
Same as above comment. All code should be prepared in the prepare section using the relevant commands so that versions get tracked in changelog by jenkins.